.github/actions/setup/action.yml

```diff
@@ -41,4 +41,4 @@ runs:
 
     - name: Install dependencies
       shell: bash
-      run: pnpm install --frozen-lockfile
+      run: pnpm install --no-frozen-lockfile
```

Comment:
I had to change that to upgrade to pnpm v9. Do we need to set it back to `--frozen-lockfile` in a follow up PR @shairez ?

Reply:
why was the change needed on CI? can you please give more context?

Reply:
There was this frozen lockfile error:

<img width="1470" alt="image" src="https://github.com/qwikifiers/qwik-ui/assets/45822175/39c7ffbb-2153-47df-b30b-78a40a4ca1a1">